data/init.sql

```diff
@@ -152,4 +152,12 @@ CREATE TABLE IF NOT EXISTS course_sections(
 	course_section_name VARCHAR(60),
 	created_at TIMESTAMP,
 	updated_at TIMESTAMP
-)
+);
+
+CREATE TABLE IF NOT EXISTS notices (
+	id INT(11) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+	notice_text VARCHAR(10000),
```

Comment:
Btw for this it's probably better to use the "text" data type

Reply:
I thought about using that but I had read that its not really an ok data type to use
but if you think it's better I'll change it

Reply:
Oh what's wrong with it?

Reply:
done

Reply:
I don't remember, but looking around quickly I can't find any counter indication so text is probably better

Reply:
OK  cool! yogesh used `text` in the tutorial too apparently.